watson-developer-cloud/discovery-components
# fix: error in document preview with split docs

#### What do these changes do/fix?

This fixes the error in rendering doc by following steps mentioned in the issue. 

PdfFallback.tsx has a check `fieldValue[fieldProp]` which was failing when `fieldValue` was undefined/null. This PR fixes the issue and adds another check. 

Related issue: https://github.ibm.com/Watson-Discovery/wd-issues/issues/1503

#### How do you test/verify these changes?

use `yarn link` -> load the discovery-tooling locally with configuring the cluster mentioned in the issue. 
(https://zen-gmc-cpd-zen-gmc.apps.tough-cookie-lb-1.fyre.ibm.com/ ) -> go to the collection `akm_test` -> search for a word `trip` -> get back results -> click on View passage in document -> You should no longer see the error `Error previewing the document`
